src/renderer/components/NewProjectModal/ModalContainer.tsx

```diff
@@ -34,36 +37,53 @@ const ModalContainer = ({ isOpen, closeModal }: Props) => {
     [dispatch]
   );
 
-  const viewComponents = [ImportMediaView, RunTranscriptionView];
+  const viewComponents = [
+    NewProjectView,
+    UploadVideoView,
```

Comment:
This was originally called `ImportMediaView`, for two reasons:
- It's not a really an upload, since there's no actual file transfer taking place - hence 'import' 
- It's not guaranteed to be a video file, it could be an audio file - hence 'media'

Reply:
refactor done in: https://github.com/patrickbrett/mlvet/pull/38/commits/40010a865dd4a23174225bea93e2d2876e78655d